readme.txt

```diff
@@ -1,12 +1,12 @@
 === WooCommerce Admin ===
-Contributors: woocommerce, automattic
+Contributors: automattic

```

Comment:
Looks like these changes might have slipped in from an old branch perhaps?

Reply:
Thanks for catching that! It looks like I checked out the changes from the old `master`, rather than `main`. Fixed in 53cbfde9